src/logic/dictionaryUtils.ts

```diff
@@ -120,7 +120,7 @@ export function matchesToCommanderHelper(
 
 /**
  * Given a collection of matches, create a dictionary of players with optional filters
- * @param matches The collection of matches to build the dictionary from
+ * @param matches The collection of matches to build the dictionary from. For stats it filters out matches with incorrect player counts
```

Comment:
Can we also move this to the method description? Can we also be a little more descriptive about what "stats" you are referring to here.

Something like: 
Given a collection of matches, create a dictionary of players with optional filters where the stats of those players are computed only using valid matches (as determined by a certain number of players in the match). 

I know it seems very verbose, but we are using specific terminology to describe exactly what is being done, as opposed to "regular english language" :)

Reply:
changed to:

_ * Given a collection of matches, create a dictionary of players with optional filters
 * where the stats of those players are computed only using valid matches
 * (as determined by the required number of players in the match)._